comments.php

```diff
@@ -25,8 +25,18 @@
 	<?php if ( have_comments() ) : ?>
 		<h2 class="comments-title">
 			<?php
-				printf( _nx( 'One thought on &ldquo;%2$s&rdquo;', '%1$s thoughts on &ldquo;%2$s&rdquo;', get_comments_number(), 'comments title', 'twentysixteen' ),
-					number_format_i18n( get_comments_number() ), get_the_title() );
+				printf(
+					/* translators: %1$s: number of comments, %2$s: post title */
```

Comment:
The placeholder should only be the number: `/* translators: 1: number of comments, 2: post title */`